src/main/java/org/datavyu/util/Subject.java

```diff
@@ -0,0 +1,26 @@
+package org.datavyu.util;
+
+import org.datavyu.plugins.ffmpeg.MediaPlayer;
+
+public interface Subject {
```

Comment:
Refactor: Subject -> Clock

Reply:
It would make sense to rename it as ExternalClock or MasterClock

Reply:
Let's use MasterClock (since you used in the some of the variable names).